cli/src/lib.rs

```diff
@@ -2235,9 +2280,18 @@ fn idl_set_buffer(
             print_idl_instruction("SetBuffer", &ix, &idl_address)?;
         } else {
             // Build the transaction.
+            let mut instructions = vec![ix];
+            if let Some(priority_fee) = priority_fee {
+                let priority_fee = get_recommended_micro_lamport_fee(&client, priority_fee)?;
+                instructions.insert(
+                    0,
+                    ComputeBudgetInstruction::set_compute_unit_price(priority_fee),
+                );
+            }
```

Comment:
This logic seems to be used multiple times, it would be great to either include this in `get_recommended_micro_lamport_fee` or create a wrapper function.

Reply:
I see this as resolved because others use `prepend_set_compute_unit_ix` but not this one. Also, the whole highlighted code from top to bottom is common, including

```rs
let mut instructions = vec![ix];
if let Some(priority_fee) = priority_fee {
    // ...
}
```